examples/How_to_use_moderation.ipynb

```diff
@@ -316,9 +330,7 @@
    "source": [
     "### 3. Custom moderation\n",
     "\n",
-    "Custom moderation involves tailoring moderation criteria and rules to the specific needs of your application, allowing for greater precision in filtering content according to unique community standards or topics. Custom moderation can present an alternative to the moderation API, as well as an extention to this set of general categories. While custom moderation requires more effort to set up and maintain, it offers a higher degree of control, making it ideal for niche platforms or specialized content where the nuances of moderation need to be finely tuned.\n",
-    "\n",
-    "Custom moderation shares the common goal of ensuring content safety and adherence to specific standards within applications with guardrails. Both involve setting criteria to filter and manage content, whether it's user input or output from language models. However, custom moderation is more focused on tailoring these criteria to the unique needs of a specific platform or community, often involving a more granular level of control. While custom moderation is a form of guardrail, guardrails can also encompass other preventive measures like rate limiting or user behavior monitoring. For those interested in exploring guardrails further, we have a cookbook that provides examples and guidance on how to use guardrails effectively, which can be found [here](https://cookbook.openai.com/examples/how_to_use_guardrails).\n",
+    "Custom moderation provides a tailored approach to content filtering, allowing for precise adherence to specific community standards or topics, making it an ideal choice for niche platforms or specialized content. Unlike the general categories covered by the moderation API, custom moderation offers a higher degree of control and requires more effort to set up and maintain. It shares the common goal of ensuring content safety with guardrails, focusing on setting criteria to filter and manage content, whether it's user input or output from language models. Custom moderation can be an alternative or an extension to the moderation API, that uses our completions endpoint. For those interested in exploring guardrails further, a cookbook with examples and guidance is available [here](https://cookbook.openai.com/examples/how_to_use_guardrails).\n",
```

Comment:
This is better, but I'd still call out explicitly (at the beginning) that unlike the prior examples, this approach **does not use the moderation api** (in bold, even).

Reply:
done!